include/alpaka/acc/AccCpuSyclIntel.hpp

```diff
@@ -7,101 +7,103 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-
 #pragma once
 
-#if defined(ALPAKA_ACC_SYCL_ENABLED) && defined(ALPAKA_SYCL_BACKEND_ONEAPI)
+#if defined(ALPAKA_ACC_SYCL_ENABLED) && defined(ALPAKA_SYCL_BACKEND_ONEAPI) && defined(ALPAKA_SYCL_ONEAPI_CPU)
 
-#include <alpaka/acc/AccGenericSycl.hpp>
-#include <alpaka/core/Concepts.hpp>
-#include <alpaka/core/Sycl.hpp>
-#include <alpaka/dev/DevCpuSyclIntel.hpp>
-#include <alpaka/dev/Traits.hpp>
-#include <alpaka/kernel/TaskKernelCpuSyclIntel.hpp>
-#include <alpaka/kernel/Traits.hpp>
-#include <alpaka/pltf/PltfCpuSyclIntel.hpp>
-#include <alpaka/pltf/Traits.hpp>
-#include <alpaka/vec/Vec.hpp>
+#    include <alpaka/acc/AccGenericSycl.hpp>
```

Comment:
Why are these spaces needed/recommended?

Reply:
This is the coding style demanded by the CRP rules / clang-format. See the [vote](https://github.com/ComputationalRadiationPhysics/contributing/pull/39) from last year.